src/utils/index.ts

```diff
@@ -1,196 +1,64 @@
-import rp from "request-promise";
 import config from "../config";
-import { BeastieLogger } from "./Logging";
+import { BeastieLogger, tryCatchLog } from "./Logging";
```

Comment:
```suggestion
import { BeastieLogger, swallowRejection } from "./Logging";
```
See ➡️ https://github.com/teamTALIMA/BeastieBot-V3/pull/13/files#r429028595